server/middleware/bookVisitSessionValidator.ts

```diff
@@ -0,0 +1,100 @@
+import { RequestHandler, Response } from 'express'
+import createError from 'http-errors'
+import paths from '../constants/paths'
+import logger from '../../logger'
+
+const journeyOrder: string[] = [
+  paths.BOOK_VISIT.SELECT_PRISONER,
+  paths.BOOK_VISIT.SELECT_VISITORS,
+  paths.BOOK_VISIT.CHOOSE_TIME,
+  paths.BOOK_VISIT.ADDITIONAL_SUPPORT,
+  paths.BOOK_VISIT.MAIN_CONTACT,
+  paths.BOOK_VISIT.CHECK_DETAILS,
+  paths.BOOK_VISIT.BOOKED,
+]
+
+export default function bookVisitSessionValidator(): RequestHandler {
+  return (req, res, next) => {
+    const { booker, bookingJourney, bookingConfirmed } = req.session
+    const { baseUrl, method, path } = req
+    const requestPath = baseUrl + path
```

Comment:
`${baseUrl}${path}` 😛

Reply:
Surprised that TypeScript / linter didn't complain about that. Fixed in upcoming PR...